src/main/java/com/endava/soa_5/exception_and_io/FileConverter.java

```diff
@@ -0,0 +1,90 @@
+package com.endava.soa_5.exception_and_io;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.compile;
+
+public class FileConverter {
+
+    private String outputFileName;
+    private final String FILE_NAME;
+    private final String RESOURCE_PATH;
+    private final List<String> SPLIT_STRING_LIST;
+    private List<String> contents;
+
+    FileConverter(String FILE_NAME) {
+        this.FILE_NAME = FILE_NAME;
+        RESOURCE_PATH = "src/main/resources/";
+        SPLIT_STRING_LIST = new ArrayList<>();
+    }
+
+    public List<String> convertFileContentIntoList() {
+        try {
+            contents = Files.readAllLines(Paths.get(RESOURCE_PATH + FILE_NAME).toAbsolutePath());
+
+        } catch (IOException ioException) {
+            System.out.println(ioException.getMessage());
+        }
+        return contents;
```

Comment:
What will happen if your program will be unable to execute line 32 ? 
You'll go to line 37, returning a null object. 
After this, somewhere you'll try to use convertFileContentIntoList().get(0) - what will be the result of this statement execution?
Think about this.

Reply:
done, the method is changed to void, and contents now not will return nullPointerException

Reply:
Done! Please check.